### ISSUE TITLE
Uncaught TypeError: Cannot read property 'substr' of undefined

### ISSUE THREAD

**Issue by dfcarpenter** [mbuscemi/elm-lens#6]
[Enter steps to reproduce:]

1. ...
2. ...

**Atom**: 1.23.1 x64
**Electron**: 1.6.15
**OS**: Mac OS X 10.13.1
**Thrown From**: [elm-lens](https://github.com/mbuscemi/elm-lens) package 0.1.1

### Stack Trace

Uncaught TypeError: Cannot read property 'substr' of undefined

```
At /Users/dan/.atom/packages/elm-lens/lib/text-editor-facade.js:27

TypeError: Cannot read property 'substr' of undefined
    at TextEditorFacade.isElmFile (/packages/elm-lens/lib/text-editor-facade.js:27:49)
    at Project.processAllElmFiles (/packages/elm-lens/lib/project.js:41:48)
    at /packages/elm-lens/lib/project.js:31:35
```

### Commands

```
```

### Non-Core Packages

```
atom-slime 2.8.0 
autocomplete-haskell 1.0.1 
elm-format 3.0.0 
elm-lens 0.1.1 
elmjutsu 7.2.1 
ide-haskell 2.2.0 
indent-detective 0.1.1 
ink 0.7.11 
julia-client 0.6.6 
language-elm 1.5.0 
language-julia 0.14.0 
language-lisp 0.2.0 
latex-completions 0.3.3 
lisp-paredit 0.6.0 
teletype 0.4.0 
tool-bar 1.1.0 
uber-juno 0.1.2 
```

**Comment by mbuscemi** [comment]
Ah! That'll teach me to assume that an Atom text editor has an associated file. I suspect this isn't blocking anyone, but if it is, let me know here. Otherwise, fix will be coming in 0.2.0.

**Comment by dfcarpenter** [comment]
Thanks! great project and thanks for all of your hard work.

**Comment by mbuscemi** [comment]
@dfcarpenter Can you please confirm that this is fixed for you in v0.2.0?
